### PR TITLE
ja: update summary translations

### PR DESCRIPTION
**Pull request by HidenoriKobayashi**
Update translations from SUMMARY.md.

Unfortunately, poedit changes the wrapping length of the po files and I could not get it back by changing the poedit setting for line length. I believe we won't get any more of these noisy diffs for my further changes this week.

`Cancellation` is the last item I modified in this PR.

### CODE REVIEW

File: po/ja.po [hunk 1]
```diff
@@ -525,14 +506,12 @@ msgid "Borrowing"
...
-#, fuzzy
 msgid "Borrowing a Value"
-msgstr "借用"
+msgstr "値の借用"
 
 #: src/SUMMARY.md src/borrowing.md src/borrowing/borrowck.md
-#, fuzzy
 msgid "Borrow Checking"
-msgstr "借用"
+msgstr "借用に関するチェック"
```
Comment (keiichiw) [inline po/ja.po:514]:
```suggestion
msgstr "借用チェック"
```

As "borrow checker" is usually translated to "借用チェッカー", "借用チェック" would be enough and simple

File: po/ja.po [hunk 2]
```diff
@@ -507,9 +489,8 @@ msgid "`Rc`"
 msgstr "`Rc`"
 
 #: src/SUMMARY.md src/smart-pointers.md src/smart-pointers/trait-objects.md
-#, fuzzy
 msgid "Owned Trait Objects"
-msgstr "トレイトオブジェクト"
+msgstr "所有トレイトオブジェクト"
```
Comment (keiichiw) [inline po/ja.po:493]:
```suggestion
msgstr "所有されたトレイトオブジェクト"
```

In my understanding, this is "Owned" + "Trait objects".

File: po/ja.po [hunk 3]
```diff
@@ -739,9 +708,8 @@ msgid "AIDL"
 msgstr "AIDL（Androidインターフェイス定義言語）"
 
 #: src/SUMMARY.md src/android/aidl/birthday-service.md
-#, fuzzy
 msgid "Birthday Service Tutorial"
-msgstr "/** 誕生日サービスのインターフェース。*/"
+msgstr "誕生日サービスのインターフェース"
```
Comment (keiichiw) [inline po/ja.po:712]: ”チュートリアル” instead of "インターフェイス"?

File: po/ja.po [hunk 4]
```diff
@@ -773,33 +740,28 @@ msgid "Changing API"
...
-#, fuzzy
 msgid "Sending Objects"
-msgstr "トレイトオブジェクト"
+msgstr "オブジェクトの送信"
 
 #: src/SUMMARY.md src/android/aidl/types/parcelables.md
-#, fuzzy
 msgid "Parcelables"
-msgstr "変数"
+msgstr "パーセラブル"
```
Comment (keiichiw) [inline po/ja.po:764]:
```suggestion
msgstr "Parcelables"
```

I think this is a technical term that we don't have no Japanese word. So, I guess keeping it in English would be more helpful
https://developer.android.com/reference/android/os/Parcelable

**Review by keiichiw — COMMENTED** [review]
Thank you for working on this!

**Comment by HidenoriKobayashi** [comment]
Thanks for the review and suggestions! Please take another look.

**Comment by HidenoriKobayashi** [comment]
@keiichiw san, could help review this again please?

**Review by keiichiw — APPROVED** [review]
Sorry for the late review. LGTM. Approved!

**Pull request merged**